packages/babel-plugin-polyfill-es-shims/test/fixtures/usage-pure/static-methods/stderr.txt

```diff
@@ -1,4 +1,4 @@
 Some polyfills have been added but are not present in your dependencies.
 Please run one of the following commands:
-	npm install --save array.from@^1.1.0 array.of@^1.0.0 is-nan@^1.2.1 math.acosh@^1.0.0 math.atanh@^1.0.0 math.cbrt@^1.0.0 math.clz32@^1.0.0 math.fround@^1.0.0 math.log1p@^1.0.1 math.sign@^2.0.0 object-is@^1.1.2 object.assign@^4.1.0 object.entries@^1.1.1 object.fromentries@^2.0.2 object.getownpropertydescriptors@^2.1.0 object.getprototypeof@^1.0.1 object.values@^1.1.1 reflect.getprototypeof@^1.0.0 reflect.ownkeys@^1.0.1 string.fromcodepoint@^1.0.0 string.raw@^1.0.1
-	yarn add array.from@^1.1.0 array.of@^1.0.0 is-nan@^1.2.1 math.acosh@^1.0.0 math.atanh@^1.0.0 math.cbrt@^1.0.0 math.clz32@^1.0.0 math.fround@^1.0.0 math.log1p@^1.0.1 math.sign@^2.0.0 object-is@^1.1.2 object.assign@^4.1.0 object.entries@^1.1.1 object.fromentries@^2.0.2 object.getownpropertydescriptors@^2.1.0 object.getprototypeof@^1.0.1 object.values@^1.1.1 reflect.getprototypeof@^1.0.0 reflect.ownkeys@^1.0.1 string.fromcodepoint@^1.0.0 string.raw@^1.0.1
+	npm install --save array.from@^1.1.0 array.of@^1.0.0 is-nan@^1.2.1 math.acosh@^1.0.0 math.atanh@^1.0.0 math.cbrt@^1.0.0 math.clz32@^1.0.0 math.fround@^1.0.0 math.log1p@^1.0.1 math.sign@^2.0.0 object-is@^1.1.2 object.assign@^4.1.0 object.entries@^1.1.1 object.fromentries@^2.0.2 object.getownpropertydescriptors@^2.1.0 object.getprototypeof@^1.0.1 object.values@^1.1.1 reflect.apply@^1.0.0 reflect.getprototypeof@^1.0.0 reflect.ownkeys@^1.0.1 string.fromcodepoint@^1.0.0 string.raw@^1.0.1
+	yarn add array.from@^1.1.0 array.of@^1.0.0 is-nan@^1.2.1 math.acosh@^1.0.0 math.atanh@^1.0.0 math.cbrt@^1.0.0 math.clz32@^1.0.0 math.fround@^1.0.0 math.log1p@^1.0.1 math.sign@^2.0.0 object-is@^1.1.2 object.assign@^4.1.0 object.entries@^1.1.1 object.fromentries@^2.0.2 object.getownpropertydescriptors@^2.1.0 object.getprototypeof@^1.0.1 object.values@^1.1.1 reflect.apply@^1.0.0 reflect.getprototypeof@^1.0.0 reflect.ownkeys@^1.0.1 string.fromcodepoint@^1.0.0 string.raw@^1.0.1
```

Comment:
Unrelated to this PR: we should come up with a webpage rendering these commands. Copying & pasting multiline text on the terminal is not good.

Reply:
The problem is that this command depends on your code, we cannot pre-write it on the website.

Usually with an update or code change you'll only need to install a small number of new polyfills.

Reply:
Oh I meant to some dynamic reports like
```
https://babel.dev/error?lz_string=(lz of missing depedencies, error kind)
```

Like AngularJS did in https://code.angularjs.org/1.8.0/docs/error/$compile/ctreq?p0=dash&p1=someDir

Reply:
Oh so we can compress the list of deps and modern terminals allow clicking on URLs :+1: